scene/animation/tween.cpp

```diff
@@ -274,7 +274,16 @@ bool Tween::step(double p_delta) {
 	}
 
 	if (!started) {
-		ERR_FAIL_COND_V_MSG(tweeners.is_empty(), false, "Tween started, but has no Tweeners.");
+		if (tweeners.is_empty()) {
+			String tween_id;
+			Node *node = get_bound_node();
+			if (node) {
+				tween_id = vformat("Tween (bound to %s)", node->is_inside_tree() ? (Variant)node->get_path() : (Variant)node->get_name());
```

Comment:
Why cast to Variant, aren't those already String?

Reply:
No, one is NodePath and other is StringName. They have different types.

Reply:
Then I would cast to String instead of Variant.

Reply:
`vformat()` takes Variant, but ok.

Reply:
String is a Variant type :)
It can take int or float if you use `%d` or `%f`, but here you're using `%s` which expects a String so I think it's good to be explicit.

Reply:
`%s` takes anything AFAIK and stringifies it. I use it interchangeably with `%d` when I don't need formatting.